### PR TITLE
Set up action

### PR DESCRIPTION
**Pull request by ekasbohm**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+      run: |  
+        git checkout -b ${{ inputs.branch-name }}
+        git config user.email "${{ github.actor }}@users.noreply.github.com"
+        git config user.name "${{ github.actor }}"
+        git commit -am ${{ inputs.commit-message }}
+        git push -u origin ${{ inputs.branch-name }}
+        echo 'module_pr_url<<EOF' >> $GITHUB_ENV
+      shell: bash
+    - name: Create Draft PR
+      run: gh pr create --title ${{ inputs.title }}  --body ${{ inputs.description }} --draft >> $GITHUB_ENV
```
Comment (ekasbohm) [inline action.yml]: Is there a way to make `--draft` conditional based on `inputs.draft`?

Comment (sumnerwarren) [reply action.yml]: Hmm... can you just add `if: inputs.draft == 'true'` and `if inputs.draft != 'true'` to each of these steps?

Comment (sumnerwarren) [reply action.yml]: I wonder if breaking the env variable assignment across steps will cause issues? Another option might be having two initial steps: `echo 'draft=' >> $GITHUB_ENV` and `if: inputs.draft == 'true', echo 'draft=--draft' >> $GITHUB_ENV` and then just include `$DRAFT` in the pr command?

Comment (ekasbohm) [reply action.yml]: > can you just add if: inputs.draft == 'true' and if inputs.draft != 'true' to each of these steps?

I tried that but my IDE was throwing "Property `if` is not allowed".  Maybe a false positive?

Comment (sumnerwarren) [reply action.yml]: Ah. Well you can probably just move that into bash then:

```
run: |
        if [ '${{ inputs.draft }}' = 'true' ]; then
          echo 'draft=--draft' >> $GITHUB_ENV
        else
          echo 'draft= ' >> $GITHUB_ENV
        fi
        git checkout -b ${{ inputs.branch-name }}
        git config user.email "${{ github.actor }}@users.noreply.github.com"
        git config user.name "${{ github.actor }}"
        git commit -am ${{ inputs.commit-message }}
        git push -u origin ${{ inputs.branch-name }}
        echo 'module_pr_url<<EOF' >> $GITHUB_ENV
        gh pr create --title ${{ inputs.title }} --body ${{ inputs.description }} $DRAFT >> $GITHUB_ENV
        echo 'EOF' >> $GITHUB_ENV
```

**Review by sumnerwarren — APPROVED** [review]
Looks good to me. Hope it works like we expect!

**Pull request merged**